x-pack/legacy/plugins/ml/public/data_frame/pages/data_frame_new_pivot/components/step_create/step_create_form.tsx

```diff
@@ -279,11 +288,11 @@ export const JobCreateForm: SFC<Props> = React.memo(
         </EuiFlexGroup>
         <EuiFlexGroup alignItems="center" style={FLEX_GROUP_STYLE}>
           <EuiFlexItem grow={false} style={FLEX_ITEM_STYLE}>
-            <EuiCopy textToCopy={getJobConfigDevConsoleStatement()}>
+            <EuiCopy textToCopy={getTransformConfigDevConsoleStatement()}>
               {(copy: () => void) => (
                 <EuiButton onClick={copy} style={{ width: '100%' }}>
                   {i18n.translate(
-                    'xpack.ml.dataframe.jobCreateForm.copyJobConfigToClipBoardButton',
+                    'xpack.ml.dataframe.stepCreateForm.copyTransformConfigToClipBoardButton',
```

Comment:
This should be `Clipboard` rather than `ClipBoard`

Reply:
Fixed in 5867350.